holocube/element/cube.py

```diff
@@ -96,7 +69,15 @@ def points(self, kdims=None, vdims=None, mdims=None, **kwargs):
 
 
 
-class CubeData(DataColumns):
+class CubeData(GridColumns):
```

Comment:
I see the super class is now ``GridColumns`` which makes sense (it defines the interface for gridded data like this). Not sure about the name ``CubeData`` although I'm not certain we can fit it into the ``*Columns`` naming scheme either...